pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1719,11 +1759,14 @@
     ::
     ?:  =(our sndr.packet)
       event-core
-    ?:  (~(has in snub.ames-state) sndr.packet)
+    =+  snubbed=(~(has in ships.snub.ames-state) sndr.packet)
+    ?:  ?|  &(=(%deny form.snub.ames-state) snubbed)
+            &(=(%allow form.snub.ames-state) !snubbed)
+        ==
```

Comment:
I would suggest this since it avoids pinning `snubbed` to the subject and also directly captures the intended xor-logic.
```suggestion
    ?:  .=  =(%deny form.snub.ames-state)
        (~(has in ships.snub.ames-state) sndr.packet))
```